content/en/blog/2022/announcing-opentelemetry-demo-release.md

```diff
@@ -0,0 +1,67 @@
+---
+title: OpenTelemetry Demo now Generally Available!
+linkTitle: Demo GA Release
+date: 2022-10-24
+author: Austin Parker
+---
+
+Earlier this year, we announced a project to build an [OpenTelemetry
+Demo](/blog/2022/demo-announcement/), representing the
+breadth of OpenTelemetry features and languages.
+Today, the Demo SIG is proud to announce [OpenTelemetry Demo
+v1.0](https://github.com/open-telemetry/opentelemetry-demo/tree/v1.0.0)!
+With this demo, you’ll be able to quickly run a complete end-to-end distributed system instrumented with 100% OpenTelemetry Traces and Metrics.
+
+![demo image](https://user-images.githubusercontent.com/47896520/196496223-6d6ea729-5bea-4a8c-a2c6-cd51cce386ae.png)
+
+One of our primary goals of this project has been to create a robust sample
+application for developers to use in learning OpenTelemetry, and we’re proud to
+say that we’ve done just that.
+Every OpenTelemetry language SDK is
+[represented](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/service_table.md)
+in this release, with the exception of Swift -- yes, even PHP. 
+We’ve built complete [tracing flows](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/trace_service_features.md) that demonstrate a breadth of common instrumentation tasks such as:
+
+- Enriching spans from automatic instrumentation.
+- Creating custom spans for richer, more useful traces.
+- Propagating trace context automatically and manually.
+- Handling observability baggage.
```

Comment:
For what purpose is baggage used here (for readers who may not totally get the concept)